platforms/web/lib/useWysiwyg.ts

```diff
@@ -71,11 +71,6 @@ export function useWysiwyg(wysiwygProps?: WysiwygProps) {
 
     const formattingFunctions = useFormattingFunctions(ref, composerModel);
 
-    const onError = useCallback(
-        (content?: string) => initModel(content),
-        [initModel],
-    );
-
```

Comment:
This was needless wrapping of initModel in another `useCallback`. This has effectively been moved inside the `useComposerModel` hook